src/commands/subcommands/github/handleAddLabels.ts

```diff
@@ -18,18 +18,38 @@ export const handleAddLabels: Subcommand = {
       const labels = interaction.options.getString("labels", true);
       const labelNames = labels.split(",").map((l) => l.trim());
 
-      await Bot.octokit.issues.addLabels({
-        owner: "freeCodeCamp",
-        issue_number: number,
-        repo,
-        labels: labelNames
-      });
+      const response = await Bot.octokit.request(
```

Comment:
We should be able to use `Bot.octokit.issues.listLabelsForRepo()` rather than constructing our own request.

Reply:
I think I got it. The method should do exactly what the custom request did.